docs/resources/dhcp_server.md

```diff
@@ -0,0 +1,31 @@
+# mikrotik_dhcp_server
```

Comment:
You probably were already thinking about this, but please update to use `tfplugindocs`.